android/src/main/java/com/oney/WebRTCModule/DailyAudioManager.java

```diff
@@ -172,35 +170,25 @@ private void transitionToCurrentCallMode(Mode previousMode) {
     }
 
     private void requestAudioFocus() {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Not sure how they ever got this building. `Build.VERSION_CODES.O` requires android sdk 26 and if that's the min sdk version then this condition will always we true